stock_move_line_lock_qty_done/models/stock_move_line.py

```diff
@@ -3,13 +3,20 @@
 
 from odoo import _, models
 from odoo.exceptions import UserError
+from odoo.tools import config
 
 
 class StockMoveLine(models.Model):
 
     _inherit = "stock.move.line"
 
     def _check_qty_done_change_allowed(self, vals):
+        check_qty_done_change_allowed = (
+            config["test_enable"]
+            and self.env.context.get("test_stock_move_line_lock_qty_done")
+        ) or not config["test_enable"]
+        if not check_qty_done_change_allowed:
+            return True
```

Comment:
Shouldn't this suffice?
```suggestion
        if config["test_enable"] and not self.env.context.get("test_stock_move_line_lock_qty_done"):
            return True
```